riptide-failsafe/src/main/java/org/zalando/riptide/failsafe/EpochSecondsDelayParser.java

```diff
@@ -0,0 +1,50 @@
+package org.zalando.riptide.failsafe;
+
+import com.google.common.base.CharMatcher;
+
+import javax.annotation.Nullable;
+import java.time.Clock;
+import java.time.Duration;
+import java.time.Instant;
+
+import static java.lang.Long.parseLong;
+import static java.time.Duration.between;
+import static java.time.Instant.now;
+
+final class EpochSecondsDelayParser implements DelayParser {
+
+    private final CharMatcher digit = CharMatcher.inRange('0', '9').precomputed();
+
+    private final Clock clock;
+
+    EpochSecondsDelayParser(final Clock clock) {
+        this.clock = clock;
+    }
+
+    @Override
+    public Duration parse(final String value) {
+        return until(parseDate(value));
+    }
+
+    @Nullable
+    private Instant parseDate(final String value) {
+        if (isInteger(value)) {
+            final long epochSecond = parseLong(value);
+            if (epochSecond > 1_000) {
```

Comment:
What about we (dynamically?) compare with `now() - '1 day'` and decide based on this whether it's seconds vs epoch timestamp? This way we'd allow servers to configure delays a la '1 hour'.